README.md

```diff
@@ -220,36 +223,4 @@ See the *LICENSE* file for more info.
 
 # Changelog
 
-### 1.x.x (YYYY-mm-dd)
-
-**SDK**
-- *Feature:* Possibility to load `testing` Mini Apps from RAS
-- *Feature:* Possibility to use back and forward navigation inside MiniApp with SDK default UI or custom client provided UI
-- *Feature:* Ability to add a custom string(RMAHostAppUserAgentInfo) that will get appended in the User agent.
-
-**Sample App**
-- *Feature:* Implemention of the `testing` Mini Apps SDK feature
-- *Feature:* Added example of custom view to navigate backward inside MiniApp
-
-**Sample App**
-- *Bugfix:* First time settings success dialog dismissed before tapping OK
-- *Bugfix:* "Display MiniApp" button was not visible when scrolling in the list Mini Apps
-
-### 1.1.1 (2020-06-11)
-
-**SDK**
-- no changes
-
-**Sample App**
-- *Bugfix:* First time settings success dialog dismissed before tapping OK
-- *Bugfix:* "Display MiniApp" button was not visible when scrolling in the list Mini Apps
-
-### 1.1.0 (2020-06-01)
-
-- Added JavaScript bridge for passing data between Mini App and Host App. Your App now must implement `MiniAppMessageProtocol` in your view controller and provide the implementation when calling `MiniApp#create`.
-- Deprecated `MiniApp#create(appInfo:completionHandler:)`. Your App should instead use `MiniApp#create(appInfo:completionHandler:messageInterface)`.
-- Added `getUniqueId` function to `MiniAppMessageProtocol`. This function should provide a unique identifier (unique to the user and device) to Mini Apps.
-
-### 1.0.0 (2020-04-27)
-
-- Initial release
+See the full [CHANGELOG](CHANGELOG.md).
```

Comment:
See my comment about the changelog on the JS SDK repo: https://github.com/rakutentech/js-miniapp/pull/25/files#r457785133

Reply:
Let's converge it to that thread.